app/controllers/katello/api/registry/registry_proxies_controller.rb

```diff
@@ -449,12 +451,19 @@ def disable_strong_params
     end
 
     def confirm_settings
-      return true if SETTINGS[:katello][:container_image_registry]
+      return true if SETTINGS.dig(:katello, :container_image_registry)
       render_error('custom_error', :status => :not_found,
                    :locals => { :message => "Registry not configured" })
       false
     end
 
+    def confirm_push_settings
+      return true if SETTINGS.dig(:katello, :container_image_registry, :allow_push)
+      render_error('custom_error', :status => :not_found,
+                   :locals => { :message => "Registry push not supported" })
+      false
```

Comment:
This line isn't necessary (it'll return nil, which is falsey), but also, I don't think this line will ever get run since it's after a render.